mixer/test/client/istio_authn/authn_report_test.go

```diff
@@ -0,0 +1,323 @@
+// Copyright 2018 Istio Authors. All Rights Reserved.
```

Comment:
Is this report only, or for both report and check. We may need want for check, as authZ will need that.

Reply:
Added the tests for check.